### ISSUE TITLE
tonsils with orange deposits

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4748]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0030814

**Synonym to be added**

tonsils with orange deposits
**broad/exact/narrow/plural/abbreviation ?**
exact

**Comment by LCCarmody** [comment]
One thing-- The term label is Orange discoloured tonsils, but this is the UK spelling. I thought American English was the standard? And if not, shouldn't there be a has_synonym_type for American Spelling? Currently, there is one for UK spelling.

**Comment by pnrobinson** [comment]
fixed the UK spelling and added other synonym.
